assets/hexagon-logo.svg

```diff
@@ -1 +1 @@
53-.045 1.055.03 1.56.165 3.862 1.036 17.16 8.234 26.534 14.36 4.347 2.843 6.422 4.462 7 5.465.703 1.217 1.145 5.037 1.407 12.147l.11 3.04-1.952 2.037a89.216 89.216 0 0 1-6.985 6.585c-1.7 1.44-5.43 4.377-5.54 4.363-.026-.003-.006-.83.046-1.836.334-6.603.028-16.263-.57-17.934-.293-.82-1.218-1.635-3.945-3.468-7.208-4.847-17.978-10.316-26.42-13.416-1.403-.515-3.996-1.343-5.763-1.84l-3.213-.903 3.558-2.013c1.957-1.108 4.06-2.28 4.67-2.602z"/><path fill="#d53b60" d="M28.448 41.302c0 .133-.047 1.4-.104 2.817-.234 5.847.018 13.706.51 15.926.305 1.367.817 1.854 4.508 4.285 10.215 6.726 21.56 12.148 32.286 15.433l2.462.755-3.537 1.91c-7.38 3.983-11.54 5.96-13.55 6.44-1.164.28-1.99.098-4.582-1.008-8.403-3.586-26.974-14.654-30.37-18.102-1.04-1.052-1.265-2.048-1.73-7.597-.133-1.58-.403-7.688-.356-8.058.003-.025.836-.924 1.85-1.997 3.106-3.283 6.464-6.305 10.503-9.45 2.202-1.716 2.113-1.66 2.11-1.353z"/><path fill="#f95" d="M86.285 48.803c.092.16-.052 7.093-.223 10.652-.27 5.658-.703 8.67-1.398 9.74-.73 1.124-4.41 3.823-9.625 7.06l-3.063 1.9-2.174-.495c-3.7-.84-9.16-2.684-13.867-4.677l-2.223-.942 6.906-3.986c6.552-3.784 7.086-4.12 10.445-6.57 6.465-4.714 11.82-9.21 14.955-12.552.116-.123.235-.18.264-.13z"/><path fill="#58c095" d="M31.11 20.73c3.35.872 6.62 1.88 9.32 2.876 2.353.867 6.073 2.403 6.032 2.49-.017.037-1.076.594-2.354 1.238-4.356 2.194-12.64 7.338-14.385 8.93-.313.288-1.46 1.17-2.545 1.96-4.184 3.047-8.304 6.518-11.52 9.706-.88.872-1.63 1.58-1.667 1.57-.112-.025.024-7.196.212-11.053.304-6.265.777-9.132 1.658-10.052 1.22-1.274 7.725-5.813 11.164-7.79l1.126-.646 2.96.77z"/><path fill-opacity=".096" d="M72.04 57.012c.254-.162.782-.575 1.883-1.433 1.67-1.3 6.188-5.155 6.188-5.155s.003-.62-.137-1.363c-1.113-5.882-3.078-11.54-3.57-13.76-.305-1.368-.818-1.857-4.51-4.287-12.343-7.21-26.914-9.895-39.7-13.287l-.042-.02 2.464.76c10.727 3.285 22.07 8.705 32.284 15.43 3.69 2.43 4.203 2.92 4.507 4.287.493 2.22.81 10.08.512 15.925-.073 1.434-.117 2.54-.115 2.948 0 .153-.014.116.24-.046z"/><path fill-opacity=".096" d="M13.994 49.498s.786-.7 1.666-1.572c3.216-3.187 7.333-6.653 11.517-9.7 1.087-.792 2.232-1.675 2.545-1.96 1.745-1.594 10.033-6.737 14.39-8.932 1.277-.644 2.333-1.2 2.35-1.238.04-.087-3.683-1.625-6.036-2.493a83.225 83.225 0 0 0-3.285-1.124c-6.004 4.45-16.273 13.552-23.146 27.018zM86.31 48.536s-.787.7-1.667 1.572c-3.216 3.188-7.333 6.654-11.517 9.7-1.087.792-2.232 1.675-2.545 1.962-1.744 1.593-10.032 6.735-14.388 8.93-1.278.644-2.334 1.2-2.35 1.238-.04.087 3.68 1.625 6.035 2.493 1.022.378 2.127.753 3.285 1.125 6.005-4.45 16.274-13.553 23.147-27.02z"/><path fill-opacity=".096" d="M28.25 41.22c-.252.163-.78.576-1.882 1.434-1.67 1.3-6.188 5.154-6.188 5.154s-.002.62.138 1.362c1.113 5.883 3.078 11.54 3.57 13.76.305 1.368.818 1.857 4.51 4.288 12.343 7.21 26.914 9.895 39.7 13.287l.042.02-2.463-.76c-10.727-3.285-22.07-8.705-32.284-15.43-3.69-2.43-4.203-2.92-4.507-4.287-.493-2.22-.81-10.08-.512-15.925.073-1.434.117-2.54.115-2.948 0-.153.013-.116-.24.046z"/></svg>
```

Comment:
There were two logos being included, 'assets/logo.svg' and 'logo.svg'
Logo.svg didn't work when the assets weren't being embedded so I've changed the assets to use 'assets/logo.svg' instead.